test/junit/src/test/scala/strawman/collection/immutable/ChampMapSmokeTest.scala

```diff
@@ -0,0 +1,170 @@
+package strawman.collection.immutable
+
+import org.junit.Assert.{assertEquals, assertFalse, assertTrue}
+import org.junit.Test
+
+object ChampMapSmokeTest {
+
+  private def emptyMap[K, V]: ChampHashMap[K, V] =
+    ChampHashMap.empty[K, V]
+
+  private def mapOf[K, V](keyValuePairs: (K, V)*): ChampHashMap[K, V] = {
+    val builder = ChampHashMap.newBuilder[K, V]()
+    keyValuePairs.foreach(builder.addOne)
+    builder.result
+  }
+
+  def tuple[KV](keyValue: KV): (KV, KV) = tuple(keyValue, keyValue)
+
+  def tuple[K, V](key: K, value: V): (K, V) = Tuple2.apply(key, value)
```

Comment:
Why don’t you just write `(key, value)` to make a tuple? Alternatively, you can use the `key -> value` syntax.